fundamentals/minimal-apis/samples/IResultImplementation/appsettings.json

```diff
@@ -0,0 +1,12 @@
+{
+  "Logging": {
+    "LogLevel": {
+      "Default": "Information",
+      "Microsoft.AspNetCore": "Warning"
+    }
+  },
+  "AllowedHosts": "*",
+  "ConnectionStrings": {
+    "IResultImplementationContext": "Server=(localdb)\\mssqllocaldb;Database=IResultImplementationContext-b71964d8-db07-43dd-9b17-5d3725d46313;Trusted_Connection=True;MultipleActiveResultSets=true"
```

Comment:
I believe you are using `InMemory` database, why do you need this?

Reply:
you're right that isn't necessary